cli/cmd/testdata/upgrade_grafana_enabled_disabled.yaml

```diff
@@ -0,0 +1,2872 @@
+---
```

Comment:
where did this file come from?  is this leftover from a merge?

Reply:
not mine. Probably a merge error.